### PR TITLE
{cae}[intel/2017a] swak4Foam v20190214

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/swak4Foam/swak4Foam-20190214-intel-2017a-OpenFOAM-4.1.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+easyblock = 'ConfigureMake'
+
+name = 'swak4Foam'
+version = '20190214'
+commit = 'e1d2358'
```
Comment (boegel) [inline easybuild/easyconfigs/s/swak4Foam/swak4Foam-20190214-intel-2017a-OpenFOAM-4.1.eb]: @smoors Any reason you're not using one of the releases? https://github.com/Unofficial-Extend-Project-Mirror/openfoam-extend-swak4Foam-dev/releases

File: easybuild/easyconfigs/s/swak4Foam/swak4Foam-20190214-intel-2017a-OpenFOAM-4.1.eb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+prebuildopts = """
+    chmod -R ug+w %(installdir)s
+    rm -rf %(installdir)s
+    mkdir -p %(installdir)s
+    source $FOAM_BASH
+    export FOAM_USER_APPBIN="%(installdir)s/bin"
+    export FOAM_USER_LIBBIN="%(installdir)s/lib"
+    export WM_NCOMPPROCS=4  # adapt to the number of cores available
+    ./Allwclean
+    ./Allwmake
```
Comment (boegel) [inline easybuild/easyconfigs/s/swak4Foam/swak4Foam-20190214-intel-2017a-OpenFOAM-4.1.eb]: @smoors Yeah, this should really be a custom easyblock... :)

File: easybuild/easyconfigs/s/swak4Foam/swak4Foam-20190214-intel-2017a-OpenFOAM-4.1.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/s/swak4Foam/swak4Foam-20190214-intel-2017a-OpenFOAM-4.1.eb]: As written here: https://openfoamwiki.net/index.php/Installation/swak4Foam/Downloading#OpenFOAM_4.0_to_4.x you need the development version for OF4.X support.
Upon closer look there are indeed new releases out that do support more recent versions of OF.
Quite confusing though: a release of a development version...

**Comment by alirezaghavaminia** [comment]
Hello, 
What happened to the swak4Foam Easyconfig. Does Easybuild support swak4Foam installation?

**Comment by boegel** [comment]
@alirezaghavaminia This pull request didn't get merged, because it was deemed too complex/hackish, and it's better to handle this via an easyblock.

The OpenFOAM easyblock is probably not a good fit here, since this easyconfig file was using the generic `ConfigureMake` easyblock.

It's better to open a new issue for this, or raise this question in the EasyBuild Slack or mailing list (see https://easybuild.readthedocs.io/en/latest/#getting-help, since following up on things in closed pull requests is difficult...)

**Pull request closed without merging**